### ISSUE TITLE
Changes in Story Chapter View [A11y] [BLOCKED]

### ISSUE THREAD

**Issue by Arjupta** [oppia/oppia-android#2327]
**Describe the bug**

Story Chapter View is not fully compatible with some users from the Accessibility point of view.

Issue - The contrast ratio of chapter_title TextView is not correct in the inactive state.

**To Reproduce**
Steps to reproduce the behaviour:
1. Start the Accessibility Scanner App on your device
2. Open the Oppia Android App and start First Test Topic Exploration
3. Switch to the Lessons Tab and Click on First Story TextView
3. Scan the screenshots with the Accessibility Scanner
4. You will see the tests that are not passed by that screen

**Expected behaviour**
All the tests must be passed by scanning the screenshots for both Tablet and Mobile Devices in every orientation.

**Screenshots**
![Screenshot_2020-12-17-11-04-48-98_80a9469f53bfe75dbc6e8989be28e2f0](https://user-images.githubusercontent.com/64064110/103155702-11a52900-47c8-11eb-896b-2e64aa2d80f1.jpg)

**Device**
- Real Device - RealMe3
- SDK version - 29

**Additional context**
The Screenshots are descriptive enough for understanding the issues but still, if there is a need of clarification then you can comment here. Multiple Pull Requests can also be made for solving these issues.

Certain Views does not have the correct contrast ratios (Inactive state) as mentioned in the screenshots. To solve this, one must find a way to make the views disabled instead of being in non-clickable state only

Reference - [https://github.com/oppia/oppia-android/wiki/Working-on-UI#accessibility-scanner](https://github.com/oppia/oppia-android/wiki/Working-on-UI#accessibility-scanner)

**Comment by Arjupta** [comment]
@rt4914 I think we can close this issue?

**Comment by rt4914** [comment]
Closing this issue as the UI has been updated.
